q2galaxy/core/templaters/helpers.py

```diff
@@ -654,7 +669,8 @@ def tests_xml(self):
                 selected_branch = galaxy_name
                 break
         else:
-            raise Exception("How did this happen?")
+            raise Exception("Argument %s is incompatible with %s" %
```

Comment:
This is a much better error messsage lol

Reply:
Yeah the other one was. . . Not helpful as we discovered when we saw it lol